src/components/EditForm.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.